workshops/templates/workshops/person_bulk_add_results.html

```diff
@@ -25,13 +25,13 @@
                 <input type="text" name="personal" value="{{ entry.personal|default:"" }}" class="hidden">
             </td>
             <td class="editable">
-                <span>{{ entry.middle|default:"&mdash;" }}</span>
-                <input type="text" name="middle" value="{{ entry.middle|default:"" }}" class="hidden">
-            </td>
-            <td class="editable">
                 <span>{{ entry.family|default:"&mdash;" }}</span>
                 <input type="text" name="family" value="{{ entry.family|default:"" }}" class="hidden">
             </td>
+            <td>
+                <span>{{ entry.username|default:"&mdash;" }}</span>
```

Comment:
Is it possible to populate the username field with the username that AMY will automatically generate?  In almost all cases, we'll want to use this (but only "almost").